### PR TITLE
Dev haowen yang

### PR DESCRIPTION
**Pull request by arthasem**

### CODE REVIEW

File: lab-exercise-1/HaowenFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class HaowenFeature {
+      public void feature() {
+          // this is a new feature
+      }
```
Comment (tnktian) [inline lab-exercise-1/HaowenFeature.java:4]: Please continue, Fighting!

File: lab-exercise-1/HaowenFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  public class HaowenFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (sRubyy) [inline lab-exercise-1/HaowenFeature.java:3]: Good job bro!

File: lab-exercise-1/HaowenFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+  public class HaowenFeature {
```
Comment (plphxwxt) [inline lab-exercise-1/HaowenFeature.java:1]: Sheesh!!!!!

Comment (YunduoLuo) [inline lab-exercise-1/HaowenFeature.java:1]: good work:)

File: lab-exercise-1/YunduoFeature.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+public class YunduoFeature {
+      public void feature() {
+          // this is a new feature
+          int x=0;
```
Comment (tnktian) [inline lab-exercise-1/YunduoFeature.java:4]: niceee bro!

**Review by sRubyy — COMMENTED** [review]
nice one let continue our work

**Review by plphxwxt — COMMENTED** [review]
Lets go

**Review by YunduoLuo — COMMENTED** [review]
nice.

**Pull request closed without merging**